### PR TITLE
Added Github and Youtube links in Activities page

### PR DESCRIPTION
**Pull request by Sahethi**
Added Youtube and Github Link to the Workshops of Activities page. The two icons are towards the right end of the Workshop title.
 
Fixed Issue #15

### CODE REVIEW

**Comment by kkothari2001** [comment]
@Sahethi could you add screenshots of the UI changes here?

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "next-transpile-modules": "^7.3.0",
     "react": "17.0.1",
     "react-dom": "17.0.1",
+    "react-facebook-login": "^4.1.1",
```
Comment (kkothari2001) [inline package.json:33]: What is this module used for? If it is not required (and was just used for testing) please remove it.

**Review by kkothari2001 — COMMENTED** [review]
Everything seems great, just check the unnecessary module issue and post the screenshots so we can make UI changes if necessary.

File: package.json (hunk 1 above)
Comment (Sahethi) [reply package.json:33]: Screenshot 2021-11-07 at 6 05 31 PM
Screenshot 2021-11-07 at 6 05 41 PM

I removed the "react-facebook-login" module, it's not required.

**Comment by Sahethi** [comment]
> @Sahethi could you add screenshots of the UI changes here?



> @Sahethi could you add screenshots of the UI changes here?
Screenshot 2021-11-07 at 6 05 31 PM
Screenshot 2021-11-07 at 6 05 41 PM

**Comment by kkothari2001** [comment]
@Sahethi pls push your latest commits here. The unnecesary module is still not going

**Comment by Sahethi** [comment]
> @Sahethi pls push your latest commits here. The unnecesary module is still not going

Hey I pushed the latest commits, please check.

**Comment by VedantParanjape** [comment]
@kkothari2001 Please check, I am merging this in 24hrs

**Pull request merged**